flytekit/bin/entrypoint.py

```diff
@@ -106,6 +129,9 @@ def _dispatch_execute(
             b: OR if IgnoreOutputs is raised, then ignore uploading outputs
             c: OR if an unhandled exception is retrieved - record it as an errors.pb
     """
+    error_file_name = _build_error_file_name()
+    worker_name = get_one_of("FLYTE_INTERNAL_WORKER_NAME", "_F_WN")
```

Comment:
@bgedik I have a slight preference for not setting the worker name in the error file if it's not a distributed task that requires multi error file aggregation. Unless we handle this in the backend, we'll otherwise see the worker name in the error message in the UI for ever task, even if the task is executed by just a single pod. I'd say this is unnecessary noise if there just is a single pod. What do you think about this?

Reply:
@fg91 Can you explain a bit more how the situation you described can happen? Right now both environment variables are set by the backend side plugin extension: https://github.com/flyteorg/flytekit/pull/2797/files/881d344e47a2618e6afc1946b56acbd760751e7b#diff-932d345851356237c198d139fa9a11d53d68257dc05387e933680ed2e764a50f

I can further anchor this variable getting set to the presence of the FLYTE_INTERNAL_DIST_ERROR_STRATEGY, but it is not going to make a difference?

Reply:
Addressed, regardless.